index.html

```diff
@@ -38,41 +44,12 @@ <h2 class="contentLink">LET'S CONNECT</h2>
 
 <section class="grid-container">
     <div class="grid-item">
-        <img id="header-img" src="./img/portfolio01.jpg" >
-            
-        <h2>Tonic</h2>
-            
-        <section class="details">
-        <ul>
-            <li>
-                <div>CANOPY</div>
-                <img src="./img/counter.png">
-                <div>Back End Dev</div>
-                <img src="./img/counter.png">
-                <div>2015</div>
-            </li>
-        </ul>
-        </section>
-
-        <p class="worksText">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-
-        <section class="worksTag">
-            <ul>
-                <li>html</li>
-                <li>css</li>
-                <li>javaScript</li>
-            </ul>
-        </section>
-            
-        <button class="worksButton">See Project</button>
-    </div>    
-
-    <div class="grid-item">
-        <img id="header-img" src="./img/portfolio02.jpg">
-            
-        <h2>Multi-Post Stories</h2>
-            
-        <section class="details">
+        <img id="header-img-l" src="./img/portfolio01.jpg" >
+        
+        <div class="postLeft">
+            <h2>Tonic</h2>
+                
+            <section class="details">
```

Comment:
- [ ] The previous reviewer mentioned that the font-size of almost all elements in your project is not the same as what you have in the Figma design template. I noticed that you are using `vw` for most of your font-size declarations, and this is estimated based on the viewing width. This is not what it should be. Kindly use the correct unit like `px` or `rem`
- [ ] Particularly, the toolbar nav-links - `Portfolio`, `About` and `contact` should have a font-size of `20px` but you have `0.51rem`. Please fix this.

Reply:
Hi @bobb-Rob! Thanks for your kind feedback! The thing is that if I use px or rem I can make it work on 768 screens but it doesn't work in wider screens, it looks like this:

![image](https://user-images.githubusercontent.com/96145777/221382976-e5ea4e22-b635-4386-bd2f-8bfa6d1d29b7.png)

Am I using it wrong?

Reply:
I tried changing the tool bar nav-links to 20px but I don't think 20 px is the correct size for this layout.

![image](https://user-images.githubusercontent.com/96145777/221386058-ea12b949-be0f-4391-9a2e-f622f9e49d40.png)